EZP-32239: Logging and cache revision (#1309)
ezsystems/developer-documentation: docs/guide/logging/logging.md

# Logging

[[= product_name =]] uses [Monolog](https://github.com/Seldaek/monolog) to log shop-related information.

By default, ERP messages and emails are stored in the database.
All other log entries are stored in `var/log/silver.eshop.log`.

All emails that are sent with [`MailHelperService`](../../api/commerce_api/helper_services/mailhelperservice.md) are logged automatically.

## Sensitive user data

Some logs can contain personal information such as User ID or password.

> By default, [[= product_name =]] does not log the User ID.

Reviewer: ```suggestion
By default, [[= product_name =]] does not log User IDs.
```

> You can change this behavior in the following configuration:

Reviewer: ```suggestion
You can change this behavior by modifying the following setting:
```
